site/js/helpers.js

```diff
@@ -9,26 +9,29 @@ const evaluation_tool_colors = {
     'warn': '#FFA500',
     'error': '#DEB887',
 };
+
+const OSM_MAX_VALUE_LENGTH = 255;
 /* }}} */
 
 // load nominatim_data in JOSM {{{
-// Using a different way to load stuff in JOSM than https://github.com/rurseekatze/OpenLinkMap/blob/master/js/small.js
+// Using a different way to load stuff in JOSM than https://github.com/vibrog/OpenLinkMap/
 // prevent josm remote plugin of showing message
-// FIXME: Warning in console. Encoding stuff.
 // eslint-disable-next-line no-unused-vars
 function josm(url_param) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', 'http://localhost:8111/' + url_param, true);      // true makes this call asynchronous
-    xhr.onreadystatechange = function () {    // need eventhandler since our call is async
-        if ( xhr.status !== 200 ) {
+    fetch(`http://localhost:8111/${url_param}`)
+        .then(response => {
+            if (!response.ok) {
+                alert(i18next.t('texts.JOSM remote conn error'));
+            }
+        })
+        .catch(() => {
             alert(i18next.t('texts.JOSM remote conn error'));
-        }
-    };
-    xhr.send(null);
+        });
 }
 // }}}
 
 // add calculation for calendar week to date {{{
+// eslint-disable-next-line no-unused-vars
```

Comment:
Delete code or prefix with `_` to make eslint happy?

Reply:
Thanks for your comments! :smiley: I'll take a look at them all.

Specifically regarding this one, the cleanest solution would definitely be to switch to ESM - I'll look into that. It's something I've had on my to-do list for a while.

Reply:
I went ahead and made the switch to ESM :tada: :exploding_head:, which also eliminated these eslint disable comments.